.github/workflows/master.yml

```
@@ -2,43 +1,58 @@
         with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Start environment
-        run: DOCKER_TAG=latest make start-containers
-      - name: Run tests
-        run: DOCKER_TAG=latest make params=-T tests
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: c26be9b36a513ce4fc9074e8071346dd
+          projectName: pyzn
+          branch:  master
```

yamllint

[warning] too many spaces after colon

(colons)